src/sidebar/services/annotations-exporter.ts

```diff
@@ -39,4 +46,58 @@ export class AnnotationsExporter {
       ) as APIAnnotationData[],
     };
   }
+
+  buildTextExportContent(
+    annotations: APIAnnotationData[],
+    groupName = '',
+    /* istanbul ignore next - test seam */
+    now = new Date(),
+  ): string {
+    const [firstAnnotation] = annotations;
+    if (!firstAnnotation) {
+      throw new Error('No annotations to export');
+    }
+
+    const { uri, title } = documentMetadata(firstAnnotation);
+    const uniqueUsers = [
+      ...new Set(
+        annotations
+          .map(annotation => username(annotation.user))
+          .filter(Boolean),
+      ),
+    ];
+
+    const dedentOptions: DedentOptions = { fullDedent: true };
+    const annotationsText = annotations
+      .map((annotation, index) =>
+        trimAndDedent(
+          `
+          Annotation ${index + 1}:
+          ${annotation.created}
+          ${quote(annotation)}
+          ${username(annotation.user)}
+          "${annotation.text}"
+          Tags: [${annotation.tags.join(', ')}]
+          `,
+          dedentOptions,
+        ),
+      )
+      .join('\n\n');
+
+    return trimAndDedent(
+      `
+      ${now.toISOString()}
+      ${title}
+      ${uri}
+      Group: ${groupName}
+      Total users: ${uniqueUsers.length}
+      Users: ${uniqueUsers.join(', ')}
+      Total annotations: ${annotations.length}
+      Total replies: ${annotations.filter(isReply).length}
+
+      ${annotationsText}
+      `,
+      dedentOptions,
+    );
```

Comment:
Content format documented in https://docs.google.com/document/d/1Xn6AazKkpUy85VUaj9v_utasexk8lIqP_1zveILq1So/edit#heading=h.g853pir10o5

I imagine there will be changes here before this feature is released to everyone.